tests/tacacs/test_jit_user.py

```diff
@@ -36,4 +36,12 @@ def test_jit_user(localhost, duthosts, ptfhost, enum_rand_one_per_hwsku_hostname
 
     res = ssh_remote_run(localhost, dutip, creds_all_duts[duthost]['tacacs_jit_user'],
                          creds_all_duts[duthost]['tacacs_jit_user_passwd'], 'cat /etc/passwd')
-    check_output(res, 'test', 'remote_user')
\ No newline at end of file
+    check_output(res, 'test', 'remote_user')
+
+def test_backward_compatibility_enable_authorization_jit(localhost, duthosts, enum_rand_one_per_hwsku_hostname, creds_all_duts,ptfhost, check_tacacs, check_tacacs_v6):
+    # Enable per-command authorization.
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+    duthost.shell("sudo config aaa authorization local")
```

Comment:
>duthost.shell("sudo config aaa authorization local") [](http://example.com/codeflow?start=4&length=52)

Does this impact other test?
What is the original pre-condition assumption of other testcases, such as  test_jit_user?
Should we recover the original expectation at the end of this test?

Reply:
Alternatively, if there is impact to other testcases, should we add similar `config aaa` to each testcase?

Reply:
This UT will not impact other test case, because because:
1. the original expectation of all existed UT are per-command authorization been disabled, which actually is 'sudo config aaa authorization local'. The UT her just make sure the backward compatibility.
2. The cleanup_tacacs method in utils.py will cleanup tacacs setting.